### PR TITLE
v1.4 Fix Developer Guide

### PR DESCRIPTION
**Pull request by guohuang88**
- Update examples in Developer Guide to match Sample Data and Sequence Diagram
- Fix Sequence Diagrams

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -630,7 +630,7 @@ The `EditEventCommand` class is responsible for editing a specific Event for a p
   `DATE_OR_DATETIME_OF_EVENT` parameter.
 * With `TIME`, an example of the `DATE_OR_DATETIME_OF_EVENT` is `20-12-2025, 12:00 - 15:00`
 * Without `TIME`, an example of the `DATE_OR_DATETIME_OF_EVENT` is `20-12-2025`
-* EditEventCommand will edit the selected `EVENT_INDEX` with a new event.
+* EditEventCommand will edit the selected `EVENT_INDEX` with a new event for the selected `PATIENT_INDEX`
```
Comment (choowengyan) [inline docs/DeveloperGuide.md]: Would it be better to have full stop after `PATIENT_INDEX ` to standardise across the DG?

**Review by choowengyan — CHANGES_REQUESTED** [review]
Overall, great changes on the sequence diagrams! Just a quick fix needed stated in the comment.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (guohuang88) [reply docs/DeveloperGuide.md]: Yes, I agree!

**Review by choowengyan — APPROVED** [review]
LGTM!

**Pull request merged**